bin/gimme-aws-creds

```diff
@@ -61,14 +67,12 @@ class GimmeAWSCreds(object):
            write_aws_creds = Option to write creds to ~/.aws/credentials
            cred_profile = Use DEFAULT or Role as the profile in ~/.aws/credentials
            aws_appname = (optional) Okta AWS App Name
-           aws_rolename =  (optional) Okta Role Name
+           aws_rolename =  (optional) AWS Role Name. 'ALL' will retrieve all roles.
     """
     FILE_ROOT = expanduser("~")
     AWS_CONFIG = FILE_ROOT + '/.aws/credentials'
 
     def __init__(self):
```

Comment:
Having a blank init might cause an error. I would do:
def __init__(self):
        pass

Reply:
Since there is no init work being done, we probably shouldn't include the __init__ function at all.

Reply:
Oh, actually, I think lines 70 and 71 should be put back in.  That's where they're getting initialized in the class, so if they get used elsewhere, the interpreter will complain.